ironfish/src/rpc/types.ts

```diff
@@ -54,3 +56,59 @@ export const RpcEncryptedNoteSchema: yup.ObjectSchema<RpcEncryptedNote> = yup
     serialized: yup.string().defined(),
   })
   .defined()
+
+export type RpcBlockHeader = {
+  hash: string
+  /**
+   * @deprecated Please use previousBlockHash instead
+   */
+  previous: string
+  sequence: number
+  previousBlockHash: string
+  difficulty: string
+  noteCommitment: string
+  transactionCommitment: string
+  target: string
+  randomness: string
+  timestamp: number
+  graffiti: string
+  work: string
+  noteSize: number | null
+}
+
+export function serializeRpcBlockHeader(header: BlockHeader): RpcBlockHeader {
+  return {
+    hash: header.hash.toString('hex'),
+    previous: header.previousBlockHash.toString('hex'),
+    sequence: Number(header.sequence),
+    previousBlockHash: header.previousBlockHash.toString('hex'),
+    timestamp: header.timestamp.valueOf(),
+    difficulty: header.target.toDifficulty().toString(),
+    graffiti: header.graffiti.toString('hex'),
+    noteCommitment: header.noteCommitment.toString('hex'),
+    transactionCommitment: header.transactionCommitment.toString('hex'),
+    target: BigIntUtils.writeBigU256BE(header.target.asBigInt()).toString(),
+    randomness: BigIntUtils.writeBigU64BE(header.randomness).toString(),
```

Comment:
I think we don't need the `BigIntUtils` here since that will convert these to Buffers. We can use `toString` directly on the BigInts like with `work`, below

Reply:
Done!